bodhi/server/validators.py

```diff
@@ -800,48 +778,60 @@ def validate_update(request, **kwargs):
         request.errors.status = HTTPNotFound.code
 
 
-def validate_update_owner(request, **kwargs):
+def common_validate(param, request):
     """
-    Make sure the referenced update owner is an existing user.
+    Common fuction to be called for validatation.
 
     Args:
+        param (string): Variable to pass as parameter.
```

Comment:
Let's make this a little more specific too - perhaps "Request parameter that references a username to be validated."""